release_build_files/readme.md

```diff
@@ -629,6 +629,7 @@ code.
 ## Release Notes
 ### Upcoming Release
 -   Changes
+    - General: Add FirebaseApp.GetApps(), to return the list of firebase app.
```

Comment:
nit: Add `FirebaseApp.GetApps(), to return the list of `firebase::App` instances.

Reply:
done